Channels/Insider/release_notes.md

```diff
@@ -1,45 +1,33 @@
-## April 18th, 2019 Insider Release ([v1.1.0837.02](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0837.02))
+## June 4th, 2019 Insider Release ([v1.1.0883.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01))
```

Comment:
The diff for this file isn't particularly useful. It's probably more useful to review the file contents directly: https://github.com/microsoft/accessibility-insights-windows/blob/d110667477c93efa8763ab4fd66f0f942cbf395a/Channels/Insider/release_notes.md

Reply:
Look good to me.